imagecontent/etc/containers/mounts.conf

```diff
@@ -0,0 +1 @@
+/run/secrets/rhsm:/run/secrets/rhsm
```

Comment:
i think my concern that this results in writing changes through to the build pod's container remains.